src/main/java/uk/gov/hmcts/reform/sendletter/controllers/SendLetterController.java

```diff
@@ -56,4 +59,21 @@ public SendLetterController(
 
         return ok().body(messageId);
     }
+
+    @GetMapping(path = "/{message-id}")
+    @ApiOperation(value = "Get letter status")
+    @ApiResponses({
+        @ApiResponse(code = 200, response = LetterStatus.class, message = "Success"),
+        @ApiResponse(code = 401, message = "Invalid service authorisation header"),
+        @ApiResponse(code = 404, message = "Letter not found")
+    })
+    public ResponseEntity<LetterStatus> getLetterStatus(
+        @PathVariable("message-id") String messageId,
```

Comment:
pretty sure we should read by PK

Reply:
👍  I think we should accept `letteruuid` instead of `messaged` as in case we receive duplicate letters we would have problems.I am going to update controller code to return `uuid` once letter is saved.

Reply:
the send endpoint returns message ID, actually. if we want PK, will have to create a separate PR for updating both endpoints. or can be done in next PR after db layer is in and this task can be completed

Reply:
I will be creating the PR for updating endpoint response for save letter

Reply:
The parameter should be called `id` or something like that, as the client is completely unaware of queue and messages. They just have an ID.